tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -106,27 +156,29 @@ sed -i "s|^;\s*app.enabled = .*|app.enabled = $DEFAULT_APP_LIST|g" /etc/tracim/d
 # TODO PA 2019-06-19 Rework the index-create part according to https://github.com/tracim/tracim/issues/1961
 # Make sure index is created in case of Elastic Search based search. (the command does nothing in case of simple search)
 cd /tracim/backend/
-tracimcli search index-create -c /etc/tracim/development.ini
-
+tracimcli_as_user "search index-create"
```

Comment:
same question here

Reply:
Can I use `tracimcli_as_user` directly in docker container if I'm using `docker exec -it tracim_container /bin/bash` or this is just available with the entrypoint ?